lib/pages/send_page.dart

```diff
@@ -226,8 +230,24 @@ class _SendPageState extends State<SendPage> {
     });
   }
 
-  bool handleExceptionResponse(txResponse) {
+  Future<bool> handleExceptionResponse(txResponse) async {
     if (txResponse == null || txResponse['success'] != true) {
+      final selectedAddress = await ReefAppState.instance.storageCtrl
```

Comment:
we should get address from txResponse if possible - it can already change this way